__tests__/main.test.js

```diff
@@ -160,32 +160,46 @@ describe('Articles', () => {
 				expect(comment.article_id).toBe(1)
 			})
 	})
-	it('POST 400 /api/articles/article_id/comments responds with bad request when poseted non-exist author name', () => {
+	it('POST 404 /api/articles/article_id/comments responds with user not found ', () => {
 		const body = {
 			body: 'We can do comments while testing do not we?',
 			author: 'iAmNotinUsers',
 		}
+		return request(app)
+			.post('/api/articles/1/comments')
+			.set('Accept', 'application/json')
+			.send(body)
+			.expect(404)
+			.then(({ body: { msg } }) => {
+				expect(msg).toBe('User not found')
+			})
+	})
+	it('POST 400 /api/articles/article_id/comments responds body is missing when body is empty but user is valid', () => {
+		const body = {
+			body: '',
+			author: 'icellusedkars',
+		}
```

Comment:
This test is not really needed and you have had to force the error. This test would be much better if the entire body key was missing and having SQL handle the error for you. You don't have to give custom errors for this. A Bad Request message will do perfectly 🙂